coalib/results/result_actions/AlternatePatchAction.py

```diff
@@ -0,0 +1,29 @@
+from coalib.results.Result import Result
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+
+
+class AlternatePatchAction(ResultAction):
+
+    SUCCESS_MESSAGE = 'Displayed alternate patch successfully.'
+
+    def is_applicable(self,
+                      result: Result,
+                      original_file_dict,
+                      file_diff_dict,
+                      applied_actions=()):
+        return result.alternate_diffs is not None
+
+    def apply(self,
+              result,
+              original_file_dict,
+              file_diff_dict):
+        """
+        (S)how Alternate Patch
+        """
+        old_diffs = result.diffs
+        result.diffs = result.alternate_diffs
```

Comment:
Updating the `result.diffs` so that if the user chooses `Apply Patch` then after this then the alternate patch will get applied.